### DISCUSSION TITLE
Tasmota with IR at Iskra AM550

### DISCUSSION THREAD

**Discussion by dbeckers99** [arendst/Tasmota#17773] [Q&A]
Hi all!

I have a big problem with my Tasmota IR implemenation. It all works fine with a smart meter counter EMH eBZD-W23. I can readout data, I can dump data with "senor53 d1".

Now I connect the IR head to another smart meter counter, an ISKRA AM550. This meter has an IR interface which is working with my "Device GmbH" WMBus SmartMeter sensor. So the IR interface IS WORKING and it is using SML, I checked.

But: with the Tasmota no dumped data is recognized or shown in the console. I tried different orientations of the IR head, different configuration on SML/OBIS, baudrate, parity, stopbits and so on. The console keeps quiet after "sensor53 d1".

Any idea how I can get this work?

**Comment by gemu2015** [comment]
do you use the latest dev version ?
sml dump was defective in an intermediate version and has only be fixed recently.

**Reply by dbeckers99** [reply]
Information output:
Program Version | 12.1.1(tasmota) -- | -- 2023-01-06T10:00:57
2_7_4_9/2.2.2-dev(38a443e) 0T00:01:33

Is that ok? And as I said: sml dump is perfect running if I use my EMH eBZD.

Maybe the ISKRA AM550 is using IEC protocol. And this isn't supported yet, right?

**Comment by gemu2015** [comment]
It has been fixed on 13 of January so your version is outdated

**Reply by dbeckers99** [reply]
I compiled the latest version 12.3.1 with GitPod and installed it:

Program Version | 12.3.1.4(tasmota) -- | -- 2023-01-23T13:52:58
2_7_4_9/2.2.2-dev(38a443e)

Same result, no dump data. Even if I choose 'r' (raw data) in settings script.

Now I'm lost again. How to get this f..king ISKRA AM550 to send at least any data?

**Comment by mgwoj** [comment]
Were you able to resolve the issue?
What is your configuration?

**Reply by dbeckers99** [reply]
No. The AM 550 is incompatible, possibly using a complete different data transmit protocol. Absolutely no data is displayed in debug mode, not even garbage.

I give up at this point.
